### DISCUSSION TITLE
Kata revision history page & 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by Yang-Wei-Ting** [codewars/codewars.com#2921] [Q&A]
Hi, I authored [a kata](https://www.codewars.com/kata/620bce448e757800242821fe) a while ago.
Now, nine out of ten tries I cannot access [the kata revision history page](https://www.codewars.com/kata/620bce448e757800242821fe/revisions) due to 500 Internal Server Error.
Perhaps I revised it too many times that getting revision history causes timeout?
If so, is it possible to squash some earlier commits?
Thanks!

(Actually, this post was posted in [the wrong place](https://github.com/codewars/content-issues/discussions/160) before, if any moderator can help me remove that one, I would be really grateful!)

**Comment by Blind4Basics** [comment]
Hi,

That error occurs way more often these last few days. But it's not related to your kata specifically. Did you try again, at different time periods in the day?

**Reply by Yang-Wei-Ting** [reply]
Sometimes it's 200, sometimes it's 500.
Right now it's 500.

**Reply by Yang-Wei-Ting** [reply]
It's 500 right now.

**Comment by hobovsky** [comment]
Created a dedicated issue: https://github.com/codewars/codewars.com/issues/2932
